psydac/linalg/stencil.py

```diff
@@ -337,6 +337,11 @@ def toarray( self, *, with_pads=False ):
         index = tuple( slice(p,-p) for p in self.pads )
         return self._data[index].flatten()
 
+    # ...
+    def toarray_local( self ):
+        idx = tuple( slice(p,-p) for p in self.pads )
+        return self._data[idx].flatten()
```

Comment:
This function contains exactly the same code as function `toarray` in the serial case (see 5 lines above). In order to avoid code duplication, can we have one function call the other?

Further, `toarray_local` needs a proper docstring.

Reply:
okay